### PR TITLE
Support for 4090231P9

### PR DESCRIPTION
**Pull request by Eagleman7**
https://github.com/Koenkk/zigbee2mqtt/issues/9745

### CODE REVIEW

**Comment by Eagleman7** [comment]
I have not tested this, tried making a custom converter but it crashed my pod because of a misconfig, so I am not sure if this is the best way to do it.

File: devices/philips.js [hunk 1]
```diff
@@ -1067,6 +1067,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LCW002', '4090231P9'],
+        model: '4090231P9',
+        vendor: 'Philips',
+        description: 'Hue Liane',
+        meta: {turnsOffAtBrightness1: true},
+        extend: hueExtend.light_onoff_brightness_colortemp_color(),
```
Comment (Koenkk) [inline devices/philips.js:1076]: Please add the color temperature range as described on https://www.zigbee2mqtt.io/advanced/support-new-devices/01_support_new_devices.html#_3-adding-converter-s-for-your-device

Comment (Eagleman7) [reply devices/philips.js:1076]: image

File: devices/philips.js [hunk 2]
```diff
@@ -1067,6 +1067,15 @@ module.exports = [
         extend: hueExtend.light_onoff_brightness_colortemp_color(),
         ota: ota.zigbeeOTA,
     },
+    {
+        zigbeeModel: ['LCW002', '4090231P9'],
```
Comment (Koenkk) [inline devices/philips.js:1071]: I don't expect both are needed, can you provide the data/database.db entry of your device?

Comment (Eagleman7) [reply devices/philips.js:1071]: {"id":6,"type":"Router","ieeeAddr":"0x001788010922a605","nwkAddr":28163,"manufId":4107,"manufName":"Philips","powerSource":"Mains (single phase)","modelId":"4090231P9","epList":[11,242],"endpoints":{"11":{"profId":260,"epId":11,"devId":269,"inClusterList":[0,3,4,5,6,8,4096,64515,768,64513],"outClusterList":[25],"clusters":{"genBasic":{"attributes":{"modelId":"4090231P9","manufacturerName":"Philips","powerSource":1,"zclVersion":2,"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210331","swBuildId":"1.88.1"}},"lightingColorCtrl":{"attributes":{"colorCapabilities":31,"colorTempPhysicalMin":153,"colorTempPhysicalMax":500}}},"binds":[],"configuredReportings":[],"meta":{}},"242":{"profId":41440,"epId":242,"devId":97,"inClusterList":[],"outClusterList":[33],"clusters":{},"binds":[],"configuredReportings":[],"meta":{}}},"appVersion":2,"stackVersion":1,"hwVersion":0,"dateCode":"20210331","swBuildId":"1.88.1","zclVersion":2,"interviewCompleted":true,"meta":{"configured":88764544},"lastSeen":1637341422247,"useImplicitCheckin":true}

**Comment by Koenkk** [comment]
Thanks!

**Comment by Eagleman7** [comment]
Thank you!

**Pull request merged**